### ISSUE TITLE
"disable" feature

### ISSUE THREAD

**Issue by Antonimo** [alvarotrigo/multiscroll.js#3]
"disable" feature would be nice, to disable, and turn off all event listeners, when needed.

**Comment by talgautb** [comment]
see here https://github.com/alvarotrigo/multiscroll.js/issues/21

**Comment by dsgnco** [comment]
I'm having an issue when using $.fn.multiscroll.build(); for click references

For instance:
$("a.gallery-click").click(function(){
     $.fn.multiscroll.destroy();
});
Then:
$("a.gallery-close").click(function(){
     $.fn.multiscroll.build();
});

This works, however when clicking the .gallery-close button, code inspector throws the error "Uncaught TypeError: ((m.event.special[e.origType] || (intermediate value)).handle || e.handler).apply is not a function"

Is there another way to go about preventing the page scroll on click?

Ideas?

**Comment by talgautb** [comment]
@dsgnco may be:
```
$("a.gallery-click").click(function(){
  $('#multiscroll').multiscroll.destroy();
});
```

**Comment by dsgnco** [comment]
@talgautb thanks, this is what I have implemented, the problem is on the click function to close, the multiscroll.build(); works but it throws an error in the web inspector "Uncaught TypeError: ((m.event.special[e.origType] || (intermediate value)).handle || e.handler).apply is not a function" and I cannot seem to locate the source of the error...

**Comment by TaraWaste** [comment]
Is it possible to disable the mousewheel functions so that slides only change when you click the nav?

**Comment by alvarotrigo** [comment]
> Is it possible to disable the mousewheel functions so that slides only change when you click the nav?

Yeah, use the function `$fn.multiscroll.setMouseWheelScrolling(false)`. (not in the docs, but its there)

**Comment by TaraWaste** [comment]
thanks, this works great however the navigation now doesnt change slides, when used on ```afterRender``` and on plugin initialization

**Comment by TaraWaste** [comment]
As a quick hack I've stopped the ```addMouseWheelHandler()``` call by commenting it out

**Comment by GaelJaboulay** [comment]
Hi there, I'd like also to stop the wheel navigation, either on the whole website, or on one specific section. I found that the addMouseWheelHandler is in the jquery.multiscroll.extensions.min.js but this is a js minified file that I was could download once the licence was paid. Any way to have a normal version of this file so that I make the wheel change? Should I "de" minify the file myself?

**Comment by alvarotrigo** [comment]
@GaelJaboulay if you want to disable the mouse wheel event just use the method `$.fs.multiscroll.setMouseWheelScrolling(false);` or ``$.fs.multiscroll.setKeyboardScrolling(false);``
